grapher/lineCharts/LineChart.tsx

```diff
@@ -700,6 +745,11 @@ export class LineChart
                 onTouchMove={this.onCursorMove}
             >
                 {clipPath.element}
+                <rect {...this.bounds.toProps()} fill="transparent">
```

Comment:
For an interactive reasoning for why this `<rect>` is needed, check out https://codesandbox.io/s/svg-pointer-events-m5hd7u.